cmd/import.go

```diff
@@ -57,39 +57,44 @@ func (cmd *ImportDataCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...
 	logger.Log.Debug(fmt.Sprintf("instanceId %s, dbName %s, schemaUri %s\n", cmd.instanceId, cmd.databaseName, cmd.schemaUri))
 
 	dbURI := fmt.Sprintf("projects/%s/instances/%s/databases/%s", cmd.project, cmd.instanceId, cmd.databaseName)
-	infoSchema, err := spanner.NewInfoSchemaImplWithSpannerClient(ctx, dbURI, constants.DIALECT_GOOGLESQL)
+	sp, err := spanneraccessor.NewSpannerAccessorClientImplWithSpannerClient(ctx, dbURI)
 	if err != nil {
-		logger.Log.Error(fmt.Sprintf("Unable to read Spanner schema %v", err))
+		logger.Log.Error(fmt.Sprintf("Unable to instantiate spanner client %v", err))
 		return subcommands.ExitFailure
 	}
 
 	switch cmd.sourceFormat {
 	case constants.CSV:
-		err := cmd.handleCsv(ctx, infoSchema)
+		//TODO: handle POSTGRESQL
+		dialect := constants.DIALECT_GOOGLESQL
+		err := cmd.handleCsv(ctx, dbURI, dialect, sp)
 		if err != nil {
 			logger.Log.Error(fmt.Sprintf("Unable to handle Csv %v", err))
 			return subcommands.ExitFailure
 		}
 		return subcommands.ExitSuccess
+	case constants.MYSQLDUMP:
+		err := cmd.handleDatabaseDumpFile(ctx, dbURI, constants.MYSQLDUMP, constants.DIALECT_GOOGLESQL, sp)
+		if err != nil {
+			logger.Log.Error(fmt.Sprintf("Unable to handle MYSQL Dump %v", err))
```

Comment:
Nit: Can we make this error message more actionable? For example, say reach out to Google or something similar.

Reply:
Done.